### ISSUE TITLE
[Accessibility] 2xMissing form label, 2xEmpty button and 1xBroken ARIA reference - Add new schedule page

### ISSUE THREAD

**Issue by cmunteanuRW** [Talleh/VolleyballPub#7]
**Steps to reproduce:**

https://github.com/user-attachments/assets/c4fa08a4-c54d-43af-9bd8-5e03542fc9cd

**Actual Result:**

![Image](https://github.com/user-attachments/assets/452c4021-db14-4243-a907-c6f029159e7d)

**Expected Result:**
No accessibility issues are present
